circle.yml

```diff
@@ -8,7 +8,7 @@ dependencies:
     - nvm install 4
     - nvm install 6
     - nvm install 8
-    - rm -rf node_modules
+    - rm -rf 'node_modules coverage .nyc_output coverage.lcov'
```

Comment:
Since this is wrapped in quotes, the `rm` program will try to delete a file called "node_modules coverage .nyc_output coverage.lcov". I suspect this is not what you had intended.